framework_tvb/tvb/interfaces/web/templates/jinja2/spatial/equation_displayer.html

```diff
@@ -22,7 +22,7 @@
     {% else %}
         <script type="text/javascript">
             $(document).ready(function () {
-                displayMessage('{{ errorMsg }}', "warningMessage");
+                displayMessage('{{ errorMsg }}', "errorMessage");
```

Comment:
Have you checked which of the 2 messages is displayed here?

Reply:
yes, the one from get_equation_chart